### PR TITLE
Submission for competition

### PR DESCRIPTION
**Pull request by EdisonLeeeee**
I want to attend this competition by this review.
Thanks.

### CODE REVIEW

**Comment by th-anderson** [comment]
This essay has already been published and is not original writing for the competition:
https://arxiv.org/abs/2003.05730

**Comment by nicodjimenez** [comment]
@EdisonLeeeee thank you for the submission!

@th-anderson you're right, but though the article is not 100% original, and should be penalized as such, it still provides a much condensed, valuable, summary of https://arxiv.org/abs/2003.05730, so can accept it with the caveat that will be disadvantaged relative to fully original submissions

File: submissions/Graph_Adversarial_Learning.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+---
+title: Adversarial Learning on Graph
+description: This review gives an introduction to Adversarial Machine Learning on graph-structured data, including several recent papers and research ideas in this field. This review is based on our paper "A Survey of Adversarial Learning on Graph".  
+author: Jintang Li
+---
+
+# Introduction
+
+Graph Neural Networks (GNNs) have received a considerable amount of attention in recent years, which has achieved excellent performance in many graph analysis tasks such as node classification, link prediction, and graph clustering. However, more and more researches have pointed out the *vulnerability* of GNNs, that is, they are easily affected by several will-designed inputs. As shown in Fig.1, slightly modifying the existence of either edges or nodes may lead to the wrong prediction of the target node. Such unreliable results provide the opportunity for attackers to exploit these vulnerabilities and restrict the applications of GNNs.
```
Comment (nicodjimenez) [inline submissions/Graph_Adversarial_Learning.md:9]: should be _well designed_ inputs (typo)

**Review by nicodjimenez — APPROVED** [review]
Thank you for your submission! Interesting stuff.

It would be great if you could provide some real world examples of GNN's and potential applications of GNN adversarial learning, for people like me that are new to the field.  I can see how it's a really interesting problem mathematically, but it does seem like this field has practical applications that aren't discussed here.  Is this work relevant to cybersecurity / cyber warfare, for example?

**Comment by EdisonLeeeee** [comment]
Hi there. 

Thanks for your feedback.

+ I am the first author of this article, and the submission is not just a "copy-and-paste. I think this submission is completely original.
+ Graph Adversarial Learning is an important branch of AI security. AI security is a new but promising field in recent years, it is not cybersecurity. To be exact, AI security should be the next generation of cybersecurity, focusing on the attack and defense against AI technology.
+ So far, the research on Graph Adversarial Learning is just emerging and has no practical application yet. Nevertheless, it has many potential application scenarios, such as recommender system. I incorporated an example into the review (https://github.com/Mathpix/spectra-review-paper-competition/pull/13).

**Pull request merged**